src/main/groovy/me/champeau/gradle/JMHPlugin.groovy

```diff
@@ -55,6 +79,12 @@ class JMHPlugin implements Plugin<Project> {
             dependencies.add(dependencyHandler.create("${JMH_ANNOT_PROC_DEPENDENCY}${extension.jmhVersion}"))
         }
 
+        /** or fix for jmh-generator-annprocess added to compile */
+        def jmhExclusions = {
```

Comment:
In case `org.openjdk.jmh:jmh-generator-annprocess` was added manually to `compile`, exclude the benchmark list generated in `compile` scope.